core/deployment/src/main/java/io/quarkus/deployment/dev/IsolatedDevModeMain.java

```diff
@@ -145,6 +147,25 @@ public void accept(String args) {
                 RuntimeUpdatesProcessor.INSTANCE.setConfiguredInstrumentationEnabled(
                         runner.getConfigValue("quarkus.live-reload.instrumentation", Boolean.class).orElse(false));
                 firstStartCompleted = true;
+
+                for (DevModeListener listener : ServiceLoader.load(DevModeListener.class)) {
+                    listeners.add(listener);
+                }
+                listeners.sort(new Comparator<>() {
```

Comment:
Really minor:
```suggestion
                listeners.sort(Comparator.comparingInt(DevModeListener::order));
```
should do the same, no?

Reply:
Yeah, we can do that :(